content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -568,6 +568,56 @@ The new Job itself will have a different uid from `a8f3d00d-c6d2-11e5-9f87-42010
 `manualSelector: true` tells the system to that you know what you are doing and to allow this
 mismatch.
 
+### Job tracking with finalizers
+
+{{< feature-state for_k8s_version="v1.22" state="alpha" >}}
+
+{{< note >}}
+In order to use this behavior, you must enable the `JobTrackingWithFinalizers`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+on the [API server](/docs/reference/command-line-tools-reference/kube-apiserver/)
+and the [controller manager](/docs/reference/command-line-tools-reference/kube-controller-manager/).
+It is disabled by default.
+
+When enabled, the control plane tracks new Jobs using the behavior described
+below. Existing Jobs are unaffected. Users don't interact with this feature.
```

Comment:
```suggestion
below. Existing Jobs are unaffected. As a user, the only difference you would see
is that the control plane tracking of Job completion is more accurate.
```
?

_(OK to merge without this change)_

Reply:
Sg. Done